selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/PythonFileEditingTest.java

```diff
@@ -92,38 +76,34 @@ public void checkLanguageServerInitialized() {
   }
 
   @Test(priority = 1)
-  public void checkMarkers() throws Exception {
+  public void checkMarkers() {
     editor.selectTabByName(PYTHON_FILE_NAME);
-    editor.deleteAllContent();
-    testProjectServiceClient.updateFile(workspace.getId(), PATH_TO_PYTHON_FILE, PYTHON_CLASS);
 
     // check warning marker message
-    editor.goToPosition(6, 53);
+    editor.goToPosition(18, 53);
     editor.typeTextIntoEditor("\n");
     editor.waitMarkerInPosition(WARNING, editor.getPositionVisible());
     editor.moveToMarkerAndWaitAssistContent(WARNING);
     editor.waitTextIntoAnnotationAssist("W293 blank line contains whitespace");
 
     // check error marker message
-    editor.goToCursorPositionVisible(1, 1);
+    editor.goToCursorPositionVisible(13, 1);
     editor.waitAllMarkersInvisibility(ERROR);
     editor.typeTextIntoEditor("c");
-    editor.waitMarkerInPosition(ERROR, 1);
+    editor.waitMarkerInPosition(ERROR, 13);
     editor.typeTextIntoEditor(Keys.DELETE.toString());
     editor.waitAllMarkersInvisibility(ERROR);
   }
 
   @Test(priority = 1)
-  public void checkAutocompleteFeature() throws Exception {
+  public void checkAutocompleteFeature() {
     editor.selectTabByName(PYTHON_FILE_NAME);
-    editor.deleteAllContent();
-    testProjectServiceClient.updateFile(workspace.getId(), PATH_TO_PYTHON_FILE, PYTHON_CLASS);
 
     // check contents of autocomplete container
-    editor.goToPosition(6, 53);
-    editor.typeTextIntoEditor("\n");
-    editor.goToPosition(7, 1);
+    editor.goToPosition(18, 53);
+    editor.typeTextIntoEditor("\n\n");
```

Comment:
Looks like these two `typeTextIntoEditor` commands can be united into the single command:
```
editor.typeTextIntoEditor("\n\nobject = MyClass()\nprint(object.");
```